packages/terra-form-validation/docs/ValidationFunctions.md

```diff
@@ -0,0 +1,68 @@
+# Validation Functions
+
+*react-final-form* gives you the ability to create your own validation functions. The key for react-final-form is that a validation function returns a blank message for when an input is invalid, and returns undefined when an input does not have any errors.
```

Comment:
The key for *react-final-form*

Reply:
What does it return if the input is valid?

Reply:
> returns undefined when an input does not have any errors.

Undefined when it doesn't have errors?

Also, its returns undefined when an input does not have any errors?